dashboard/db/migrate/20160809170542_add_view_answers_unlocked_at_to_user_levels.rb

```diff
@@ -0,0 +1,6 @@
+class AddViewAnswersUnlockedAtToUserLevels < ActiveRecord::Migration
+  def change
+    add_column :user_levels, :view_answers, :boolean
+    add_column :user_levels, :unlocked_at, :datetime
```

Comment:
Another possible change might be adding a `total_time` column. Today, we track (client-side) elapsed time on a level, saving it to `activities.time`. If we deprecate the `activities` table, I imagine there being value in having `user_levels.total_time`. What do you think @joshlory @ryansloan @Hamms?